packages/core/echo/live-object/src/schema.ts

```diff
@@ -0,0 +1,52 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import { Schema as S } from '@effect/schema';
+
+import {
+  createJsonSchema,
+  EchoObject,
+  getObjectAnnotation,
+  MutableSchema,
+  ObjectAnnotationId,
+  StoredSchema,
+  toJsonSchema,
+  type ObjectAnnotation,
+} from '@dxos/echo-schema';
+
+import { create, type ReactiveObject } from './object';
+
+/**
+ * Create ECHO object representing schema.
+ */
+export const createStoredSchema = ({
+  typename,
+  version,
+  jsonSchema,
+}: Pick<StoredSchema, 'typename' | 'version'> &
```

Comment:
reconcile StoredSchema and ObjectAnnotation which have the same props (see function below).